app/src/main/res/values/styles.xml

```diff
@@ -20,6 +20,11 @@
         <item name="android:textSize">@dimen/actual_temp_text_size</item>
     </style>
 
+    <style name="LoadingBarStyle">
+        <item name="android:layout_height">@dimen/loading_bar_text_height</item>
```

Comment:
as a good practice do not put the android:layout_height and android:layout_width into a style

Reply:
Thanks for the advice